src/cozmo/action.py

```diff
@@ -189,17 +189,17 @@ def _recv_msg_robot_completed_action(self, evt, *, msg):
             logger.warn('Received "running" action notification for action=%s', self)
             self._set_failed('running', 'Action was still running')
 
-        elif result == types.NOT_STARTED:
+        elif result == types.FAILURE_NOT_STARTED:
```

Comment:
Hmmm, looks like @alchaussee 's matching change for cozmo-one didn't go in? In which case there will also be runtime python issues on aborting or failing actions - how far off is that cozmo-one change from going in (these should really go in at the same time - ultimately we'll probably need to use a sub-module pointer into specific commits in nextclad), otherwise we should probably revert that earlier commit and re-apply it once it's ready.